tests/models/test_world_coordinates.py

```diff
@@ -78,6 +82,13 @@ def test_world_coordinates( ztf_datastore_uncommitted, provenance_base, provenan
                 session.commit()
             session.rollback()
 
+            # ensure you cannot overwrite when explicitly setting overwrite=False
+            with pytest.raises(
+                OSError,
+                match=".txt already exists"
+            ):
```

Comment:
You can put these arguments all on one line, it should easily fit into 120 characters...